CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 23.2.0 [#675](https://github.com/openfisca/openfisca-core/pull/675)
```

Comment:
As a reuser of Core, do I really get a new feature out of this change? It seems that this feature is only targeted to Core developpers, so I'd go for `23.1.3`.

Reply:
SemVer is not strict on that: the engagement is only that **if** there is a breaking change, you'll bump major. You're always free to bump minor for a patch change. In this case, if a country developer can now measure performance, I'd say it's definitely a new feature 🙂